tests/event-processing-tests/run-tests.sh

```diff
@@ -4,15 +4,15 @@ set -eu
 
 gradle -v
 
-cd /
+#cd /
 
 mkdir $TEST_ENVIRONMENT
 
 echo "Environment: $TEST_ENVIRONMENT"
 
 echo "Current Working Directory: $PWD"
 
-# gradle -q test --info --stacktrace
+ gradle -q test --info --stacktrace
```

Comment:
the tests were disabled as there are access control issues. as we are migrating to the self serve tool, the integration tests will also be migrated over. please leave line commented